magefiles/docker.go

```diff
@@ -1,5 +1,9 @@
 package main
 
+// NOTE: Do not import code from the nordvpn-linux app packages here.
+// It will force `mage` to look for *.so dependencies needed for the app
```

Comment:
This is only true when building packages that contain code which is using native dependencies. For most of packages in the repo this is not true.

Reply:
Done :+1: